src/AsyncGenerator.yml

```diff
@@ -5,6 +5,9 @@
   applyChanges: true
   analyzation:
     methodConversion:
+    - conversion: Ignore
+      name: GetEnumerable
```

Comment:
Why?

Reply:
Because proper async enumeration is not supported by Async Generator (and I guess won't be supported until C#8 with its `IAsyncEnumerable`). Generated code yields all results to `List` which kills the whole point of Enumerable call:
```C#
private async Task<IEnumerable<object>> GetEnumerableAsync(QueryParameters queryParameters, IEventSource session, CancellationToken cancellationToken)
{
	cancellationToken.ThrowIfCancellationRequested();
	var yields = new List<object>();
	foreach (var t in Translators)
	{
		foreach (object obj in await (t.GetEnumerableAsync(queryParameters, session, cancellationToken)).ConfigureAwait(false))
		{
	yields.Add(obj);
		}
	}
	return yields;
}
```

Besides current `EnumerableAsync` methods are not async at all - the only async part is opening reader in query loader but iteration is synchronous.